orbit/pkg/table/extension.go

```diff
@@ -95,7 +96,7 @@ func (r *Runner) Execute() error {
 
 	plugins := OrbitDefaultTables()
 
-	plugins = append(plugins, PlatformTables()...)
+	plugins = append(plugins, PlatformTables(r)...)
```

Comment:
This was left over from me wanting to pass a kitlog logger object and the path of the osquery extension unix socket. Only the path is currently used but I wanted to keep passing the whole runner because it makes it easier to add new fields in the future and the object already exists.

Reply:
We should not let one object "know" about another unless this is the design and there is a need to know.
Better to only pass down the needed info.

If you feel you want to pass down a set of field that have some relation, you could always create an object for them.
I am not 100% familiar with Runner's functionality but don't think you need it passed down.

Reply:
I just changed it to only pass  `osquerydPath`, though I do still need to add `osquerydPath` as a field in `Runner` because passing arguments directly to `r.Execute` is hard